src/extension.ts

```diff
@@ -175,8 +181,10 @@ function askForUserInputs(useUrl: boolean): Promise<any> {
 function callWsdl2Rest(wsdl2restExecutablePath: string): Promise<boolean> {
 	return new Promise( (resolve, reject) => {
 		try {
-			let storagePath: string = vscode.workspace.rootPath; // is undefined for some unknown reason
-			if (!storagePath) {
+			if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders[0]) {
+				storagePath = vscode.workspace.workspaceFolders[0].uri.fsPath;
```

Comment:
does it mean that the code while that the temporary generated code while be created inside the user workspace? Isn't a risk that it overlaps with existing real code?

Reply:
If that's a concern, we can handle it in a different PR. This is the way the code has run since the beginning and has nothing to do with this particular change.

Reply:
ah ok. Then I don't understand what is the difference between vscode.workspace.rootPath and vscode.workspace.workspaceFolders[0].uri.fsPath ? You mean that they both have the same value?

Reply:
https://code.visualstudio.com/updates/v1_38#_deprecated-workspacerootpath

Reply:
And yes. :)